docs/sources/panels/panel-editor.md

```diff
@@ -0,0 +1,78 @@
++++
+title = "Panel editor"
+type = "docs"
+[menu.docs]
+identifier = "panel-editor"
+weight = 200
++++
+
+# Panel editor
+
+This page describes the parts of the Grafana panel editor and links to where you can find more information.
+
+{{< docs-imagebox img="/img/docs/panel-editor/panel-editor-7-0.png" class="docs-image--no-shadow" max-width="1500px" >}}
+
+## Open the Panel editor
+
+There are several ways to access the panel editor, also called the **Edit Panel** screen, _edit mode_, or _panel edit mode_:
+
+- Click the **Add panel** icon at the top of the screen and then click **Add new panel**. The new panel opens in the panel editor. For detailed instructions on how to add a panel, refer to [Add a panel]({{< relref "add-a-panel.md" >}})
+- Click the title of an existing panel and then click **Edit**. The panel opens in edit mode.
+- Click anywhere on an existing panel and then press **e** on your keyboard. The panel opens in edit mode.
+
+## Resize panel editor sections
+
+Drag to resize sections of the panel editor. If the Panel, Field, and Overrides tabs become too narrow, then the tabs convert to a dropdown list.
+
```

Comment:
```suggestion
Drag to resize sections of the panel editor. If the side pane becomes too narrow, then the Panel, Field, and Overrides tabs change to a dropdown list.

```

Reply:
We're using "side pane" in documentation? Just checking as so far we were referring to it as "sidebar".

Reply:
Personally I prefer "side pane", think this is what we used in other docs too so far

Reply:
Also, "sidebar" usually refers to the icons on the left of the dashboard.